Modules/commands/_command.py

```diff
@@ -0,0 +1,160 @@
+"""
+_command.py - {summery}
+
+{long description}
+
+Copyright (c) 2018 The Fuel Rats Mischief,
+All rights reserved.
+
+Licensed under the BSD 3-Clause License.
+
+See LICENSE.md
+"""
+from collections import abc
```

Comment:
I believe i did it this way To make it clearer `Container` comes from `abc`.

Reply:
I agree, but abc is a rather large class.

Reply:
by class you mean `module`.
And even if i did `from collections.abc import Container` the memory is still consumed...

i find 
```py
class Foo(Container):
   ...
```
to be less clear than
```py
class Foo(abc.Container):
   ...
```

or would it be preferable to do
```py
import collections

class Foo(collections.abc.Container):
    ...
```
?

Reply:
Perhaps in the end, it comes down to personal preference.  In this case, I would like to see the second example (abc.Container) only to differentiate the one used from the standard library and not a custom implementation or from a third party.  Same goes with pydle's Client.  it could be httpclient, sqlclient, tcpclient or even serverclient.

This only serves for readability for someone new to the codebase.

Reply:
Which is pretty much my point. I did it this way to make it more clear what `Container` class was being inherited from. 

Since we appear to agree, il mark this conversation as resolved.